### ISSUE TITLE
std.json.JSONException@std/json.d(494): Key not found: lastModifiedDateTime

### ISSUE THREAD

**Issue by hacker1024** [skilion/onedrive#380]
Soon after starting the service, I get this error, and the service stops:
```
std.json.JSONException@std/json.d(494): Key not found: lastModifiedDateTime
----------------
??:? pure @safe void std.exception.bailOut!(std.json.JSONException).bailOut(immutable(char)[], ulong, const(char[])) [0x40ef56b3]
??:? pure @safe inout(std.json.JSONValue)* std.exception.enforce!(std.json.JSONException).enforce!(inout(std.json.JSONValue)*).enforce(inout(std.json.JSONValue)*, lazy const(char)[], immutable(char)[], ulong) [0x40f8402a]
??:? inout pure ref @safe inout(std.json.JSONValue) std.json.JSONValue.opIndex(immutable(char)[]) [0x40f3aad5]
??:? itemdb.Item sync.makeItem(ref const(std.json.JSONValue)) [0x40f0ceed]
??:? void sync.SyncEngine.applyDifference(std.json.JSONValue, immutable(char)[], bool) [0x40f0ddca]
??:? void sync.SyncEngine.applyDifferences(immutable(char)[], const(char)[]) [0x40f0dc0f]
??:? void sync.SyncEngine.applyDifferences() [0x40f0d8ed]
??:? void main.performSync(sync.SyncEngine) [0x40efc903]
??:? _Dmain [0x40efc460]
Key not found: lastModifiedDateTime
```

**Comment by abraunegg** [comment]
This is fixed by PR #314, but also detailed in #372, #351, #337 

Additionally, the 'skilion' code appears to have been abandoned by the original author and contains issues that have not been fixed. Any issues, suggestions & feature requests will not be looked at under this github repository.

If you wish to use the latest code which contains numerous fixes and enhancements you need clone from here: https://github.com/abraunegg/onedrive

If you have any issues when using my branch submit your issue against my branch for resolution & management.

**Comment by skilion** [comment]
Fixed in 19793c854d9dd97eb5accf94d21406f825c75fb5

**Comment by Mattia98** [comment]
The Ubuntu package is not yet updated to the latest version. Is there a workaround for the meantime?
